### ISSUE TITLE
As a developer, I want to know that component dependencies are missing from the cluster.

### ISSUE THREAD

**Issue by JohanvdWest** [happner/happner-cluster#147]
Currently if a subscription can't find a version that matches the requirement in package.json, there is not indication that the system is not working as expected.

We need to at least log an error message that this is the case.
We could also consider logging a message that the subscribe is successful if it is able to subscribe when a correct version comes online.

**Comment by nomilous** [comment]
Adjusted title.

When a dependency goes/is missing cluster node will log error "componentX n.n.n is missing dependency componentY ^n.n.n" 

When dependency arrives/returns "componentX n.n.n has available dependency componentY ^n.n.n"

**Comment by nomilous** [comment]
fixed in + happner-client@3.0.1 (will update in cluster per ^ in dependencies)

example logs
```
[ERROR] - 245ms	MESH_0 (HappnerClient) dependent local-component has 0 of remote-component ^1.0.0

[ INFO] - 53ms	MESH_0 (HappnerClient) dependent local-component has 1 of remote-component ^1.0.0
```
